.github/workflows/pmem_tests.yml

```diff
@@ -49,8 +49,8 @@ jobs:
   # Static builds are tested in this limited scope only.
   static:
     name: Static
-    if: github.repository == 'pmem/pmdk'
```

Comment:
${{ github.event.repository.fork == false }}

Reply:
On `schedule` it is unavailable.

Reply:
Done.